### PR TITLE
Superadmin add admin feature

### PR DESCRIPTION
**Pull request by youngGaf**
Superadmin add admin
- superadmin create admin
- setup mailing utils file to send admin details on after creating
- updated .example.env file
Screenshot 2020-08-06 at 12 14 26

### CODE REVIEW

**Comment by youngGaf** [comment]
Screenshot 2020-08-06 at 12 14 34

File: utils/password-hash.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const bcrypt = require('bcryptjs');
+
+exports.passwordHash = (password) => {
```
Comment (tonymj76) [inline utils/password-hash.js:3]: we are not adding salt here check how admin are do login and fix this code else this new admin can't login

File: utils/password-hash.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const bcrypt = require('bcryptjs');
```
Comment (tonymj76) [inline utils/password-hash.js:1]: no need of this

File: controller/adminController.js [hunk 3]
```diff
@@ -69,7 +62,52 @@ const logout = (req, res) => {
...
+    const {
+      firstName, lastName, email, password, category
+    } = req.body;
+
+    const adminExists = await Admin.findOne({ email });
+    if (adminExists) {
+      return responseHandler(res, 'Admin with that email already exist', 401, false);
+    }
+
+    const hashedPassword = await passwordHash(password);
```
Comment (tonymj76) [inline controller/adminController.js:76]: use the normal password hash

**Review by tonymj76 — CHANGES_REQUESTED** [review]

File: utils/password-hash.js (hunk 1 above)
Comment (youngGaf) [reply utils/password-hash.js:3]: salt is for database

Comment (youngGaf) [reply utils/password-hash.js:3]: Okay I get you now

**Review by tonymj76 — APPROVED** [review]

**Pull request merged**